.bowerrc

```diff
@@ -1,3 +1,3 @@
 {
-    "directory": "hu_web/hu_browser/static/hu_browser/bower/"
+    "directory": "hu_web/common-static/bower/"
 }
```

Comment:
No, from my understanding, it doesn't apply here.  The bower managed files will only ever be at the top level.  If we do want to put any additional files in there (which I was thinking of doing), we would namespace them alongside the bower subfolder.